src/MetaMorpho.sol

```diff
@@ -286,10 +286,10 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
     /// @inheritdoc IMetaMorphoBase
     function submitMarketRemoval(Id id) external onlyCuratorRole {
         if (config[id].removableAt != 0) revert ErrorsLib.AlreadyPending();
+        if (pendingCap[id].validAt != 0) revert ErrorsLib.PendingCap();
+        if (config[id].cap != 0) revert ErrorsLib.NonZeroCap();
```

Comment:
Now the process to remove a market when there's a pending cap is:
1. `revokePendingCap`
2. `submitCap(0)`
3. `submitMarketRemoval()`

This must clearly explained in the README IMO